components/AddedExercisesList/AddedExercise/openCard/AddedSets/index.js

```diff
@@ -1,9 +1,21 @@
-export default function AddedSets({ addedSet }) {
+import useSWR from "swr";
+
+export default function AddedSets({ id }) {
+  const { data, isLoading } = useSWR(`/api/db/${id}`);
+
+  if (isLoading) {
+    return <h1>Loading...</h1>;
+  }
+
+  if (!data) {
+    return;
+  }
```

Comment:
There are two further options here:
You can additionally destructure `error` from useSWR and add that to the if statement for (isLoading), or you can add !data to the same statement. Here the first example:
```suggestion
  const { data, isLoading, error } = useSWR(`/api/db/${id}`);

  if (isLoading || error) {
    return <h1>Loading...</h1>;
  }
```